code/script.js

```diff
@@ -2,324 +2,497 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const filterButton = document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose')
+const winOrLoseText1 = document.getElementById('winOrLoseText1')
+const winOrLoseText2 = document.getElementById('winOrLoseText2')
+const playAgain = document.getElementById('playAgain')
+const winOrLoseContainer = document.getElementById('winOrLoseContainer')
+const countUpTimer = document.getElementById('count-up')
+const guessCounterElem = document.getElementById('guess-counter');
+
 
-// Array with all the characters, as objects
 const CHARACTERS = [
```

Comment:
Looks like you had a lot of fun with the characters, and it definitely makes the game more enjoyable.